1/ToDoApp/src/components/ToDoList/ToDoList.spec.js

```diff
@@ -0,0 +1,7 @@
+import ToDoList from './ToDoList.vue'
+
+describe('ToDoList', () => {
+  it('renders 10 initial example-items on the list', () => {
+    expect(ToDoList.data().todos.length).toBe(10)
```

Comment:
Better expect some html here. Or simply do:

```suggestion
import ToDoList from './ToDoList.vue'
import ToDoItem from '../ToDoItem/ToDoItem.vue'

describe('ToDoList', () => {
  it('renders 10 initial example-items on the list', () => {
    wrapper = // .. setup
    expect(wrapper.findAll(TodoItem)).toHaveLength(10)
  })
})
```
See:
https://jestjs.io/docs/en/expect#tohavelengthnumber
https://vue-test-utils.vuejs.org/api/wrapper/#findall